units/src/iosMain/kotlin/dev/icerock/moko/units/UnitsSource.kt

```diff
@@ -0,0 +1,48 @@
+package dev.icerock.moko.units
+
+import dev.icerock.moko.units.TableUnitItem
+import dev.icerock.moko.units.CollectionUnitItem
+import kotlinx.cinterop.ExportObjCClass
+import kotlinx.cinterop.ExternalObjCClass
+import kotlinx.cinterop.ObjCObjectMeta
+import platform.UIKit.UICollectionView
+import platform.UIKit.UITableView
+import platform.UIKit.UICollectionViewDataSourceProtocol
+
+interface TableUnitsSource {
+    var unitItems: List<TableUnitItem>?
+    val tableView: UITableView?
+}
+
+interface CollectionUnitsSource  {
```

Comment:
i think sources must be separated to:
`TableUnitsSource.kt` and `CollectionUnitsSource.kt` with interface and `create` function